content/post/know-your-aws-community-day-india-2020-speaker-swapnil-jakhetia.md

```diff
@@ -0,0 +1,20 @@
++++
+author = "Dhruv Aggarwal"
+date = 2020-09-23T14:12:08Z
+description = "Swapnil Jakhetia will be speaking on \"Infrastructure is Code with AWS CDK\" at <time> on <date> at AWS Community Day India 2020."
+feature_image = "/images/swapnil.PNG"
+tags = ["AWS", "AWSCommunityDayIndia", "AWS CDK", "Speaker"]
```

Comment:
looks like the theme shows only the first tag, so please keep "Speaker" as the first tag in the list,

Reply:
Done!